examples/src/index.js

```diff
@@ -57,17 +43,33 @@ function HitTestExample() {
   return <Box ref={ref} args={[0.1, 0.1, 0.1]} />
 }
 
-function App() {
+function ARExample() {
   return (
-    <VRCanvas>
-      <ambientLight intensity={0.5} />
-      <pointLight position={[5, 5, 5]} />
+    <Canvas frameloop="never" onCreated={({ gl }) => startXRloop(gl)}>
+      <XR buttonAR={true} sessionInit={{ requiredFeatures: ['hit-test'] }}>
+        <ambientLight intensity={0.5} />
+        <pointLight position={[5, 5, 5]} />
+        <HitTestExample />
+      </XR>
+    </Canvas>
+  )
+}
 
-      <Button position={[0, 0.8, -1]} />
-      <DefaultXRControllers />
-      {/* <HitTestExample /> */}
-    </VRCanvas>
+function VRExample() {
+  return (
+    <Canvas frameloop="never" onCreated={({ gl }) => startXRloop(gl)}>
```

Comment:
we shouldn't require users to do `frameloop="never" onCreated={..}`, we can make a life a bit easier by providing component that already have all of that done